.github/workflows/test_trusted.yaml

```diff
@@ -62,6 +62,7 @@ jobs:
 
       - name: Run integration tests
         env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this passes the token, but it's not yet authorized for write access to issues, so the plugin will just log an exception for now